guides/notifications.md

```diff
@@ -218,6 +218,25 @@ A Dockerfile build has failed
 }
 ```
 
+#### <i class="fa fa-lg fa-times-circle-o event-icon"></i>Vulnerability Found
+<a name="#vulnerability_found"></a>
+
+Clair has discovered a new vulnerability
```

Comment:
Yeah, this should be "Quay Security Scanner has discovered a new vulnerability *or* updated and existing vulnerability to a higher severity"